inginious/common/tasks.py

```diff
@@ -81,7 +78,11 @@ def input_is_consistent(self, task_input, default_allowed_extension, default_max
 
     def get_order(self):
         """ Get the position of this task in the course """
-        return self._order
+        tasks_id = self._course.get_toc().get_tasks()
```

Comment:
There are still some parts of the code that only rely on the ``common`` files and not the ``frontend`` ones. So calling ``get_toc`` which is defined in ``FrontendCourse`` won't work in some rare cases. But actually I think the order field is only used by the frontend. So this function can be moved into ``FrontendTask``

Reply:
Indeed thanks. It's done